src/extra/others/ime/lv_ime_pinyin.c

```diff
@@ -429,6 +450,30 @@ void lv_ime_pinyin_set_dict(lv_obj_t * obj, lv_pinyin_dict_t * dict)
     init_pinyin_dict(obj, dict);
 }
 
+/**
+ * Set mode, 26 key input(k26) or 9 key input(k9).
+ * @param obj  pointer to a Pinyin input method object
+ * @param en   1: 26 key input(k26), 0: key input(k9)
+ */
+void lv_ime_pinyin_set_mode(lv_obj_t * obj, bool en)
```

Comment:
I suggest using an `enum` here.

Reply:
This proposal is very good. I will revise it soon.